pyaerocom/units_helpers.py

```diff
@@ -76,6 +90,25 @@ def unit_conversion_fac_custom(var_name, from_unit):
                                   .format(from_unit, var_name))
     return (info.to, info.fac)
 
+def unit_conversion_non_si_fac(var_name, from_unit):
+    """Get custom conversion factor for a certain unit"""
+    # if from_unit in UALIASES:
+    #     from_unit = UALIASES[from_unit]
+    try:
+        info = NON_SI_UCONV_MUL_FACS.loc[(var_name, str(from_unit)), :]
+        if not isinstance(info, pd.Series):
+            raise Exception('Could not find unique conversion factor in table '
+                            'UCONV_MUL_FACS in units_helpers.py. Please check '
+                            'for dulplicate entries')
+    except KeyError:
+        raise UnitConversionError('Failed to convert unit {} (variable {}). '
+                                  'Reason: no custom conversion factor could '
+                                  'be inferred from table '
+                                  'pyaerocom.units_helpers.UCONV_MUL_FACS'
+                                  .format(from_unit, var_name))
+    return (info.to, info.fac)
+
+
```

Comment:
I am not sure I understand why `NON_SI_UCONV_MUL_FACS` is defined and why this function is needed? Shouldn't that just be added to `UCONV_MUL_FACS`?

Reply:
Again, I confused things here!